src/client/pages/EmailSentPage.tsx

```diff
@@ -6,17 +6,24 @@ import { addQueryParamsToPath } from '@/shared/lib/queryParams';
 
 export const EmailSentPage = () => {
   const clientState: ClientState = useContext(ClientStateContext);
-  const { pageData = {}, queryParams } = clientState;
+  const { pageData = {}, queryParams, globalMessage = {} } = clientState;
   const { email, previousPage } = pageData;
+  const { emailSentPage } = queryParams;
+  const { error } = globalMessage;
 
-  const queryString = addQueryParamsToPath('', queryParams);
+  const queryString = addQueryParamsToPath('', queryParams, {
+    emailSentPage: true,
+  });
 
   return (
     <EmailSent
       email={email}
       previousPage={previousPage}
       resendEmailAction={previousPage}
       queryString={queryString}
+      showSuccess={!!emailSentPage}
```

Comment:
just a more general question: what does `emailSentPage` represent here to indicate success? (I'm sure it does, but it's not clear in my head)